tools/spec2vec/.shed.yml

```diff
@@ -6,13 +6,19 @@ categories:
 repositories:
   spec2vec_training:
     description: "Train a Spec2Vec model for mass spectra similarity scoring."
```

Comment:
Why is this file not a suite file?

Reply:
Changed in https://github.com/RECETOX/galaxytools/pull/413/commits/d2283c09b963db74b8a5a74d2bcef3429fc951b8